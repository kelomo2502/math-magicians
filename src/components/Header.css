.links {
  display: flex;
  justify-content: flex-end;
}

.link-item {
  padding: 0 8px;
  border-left: 2px solid;
}

.first-item {
  border: 0;
}
