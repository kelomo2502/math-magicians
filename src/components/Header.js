import './Header.css';

const Header = () => (
  <header className="navbar">
    <h1> Math Magicians </h1>
    <div className="links">
      <a href="/" className="link-item">
        Home
      </a>
      <a href="/calculator" className="link-item">
        Calculator
      </a>
      <a href="/quote" className="link-item">
        Quote
      </a>
    </div>
  </header>
);

export default Header;
