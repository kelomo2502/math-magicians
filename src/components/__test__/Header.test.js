import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

it('renders when there is item', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
