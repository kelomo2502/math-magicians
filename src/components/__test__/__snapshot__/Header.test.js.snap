// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the snapshot 1`] = `
 <header className="header">
      <h1 className="title">Math Magician</h1>
      <ul className="links">
        <li className="link-item first-item">
          <Link
            to="/"
            className={link === '/' ? 'link selected' : 'link'}
            onClick={() => updateLink('/')}
          >
            Home
          </Link>
        </li>
        <li className="link-item">
          <Link
            to="/calculator"
            className={link === '/calculator' ? 'link selected' : 'link'}
            onClick={() => updateLink('/calculator')}
          >
            Calculator
          </Link>
        </li>
        <li className="link-item">
          <Link
            to="quote"
            className={link === '/quote' ? 'link selected' : 'link'}
            onClick={() => updateLink('/quote')}
          >
            Quote
          </Link>
        </li>
      </ul>
    </header>`;